quarkus.ls/com.redhat.quarkus.ls/src/test/java/com/redhat/quarkus/services/ApplicationPropertiesDiagnosticsTest.java

```diff
@@ -32,17 +31,17 @@
 	@Test
 	public void validateUnknownProperties() throws BadLocationException {
 		String value = "# quarkus.datasource.url=jdbc:postgresql:quarkus_test\n" + //
-				"quarkus.datasource.driver=org.postgresql.Driver\n" + //
-				"quarkus.datasource.username=quarkus_test\n" + //
-				"quarkus.datasource.password=quarkus_test\n" + //
+				"quarkus.datasource.driver=\"org.postgresql.Driver\"\n" + //
```

Comment:
My mistake, I must remove the quotations for these test cases.

Reply:
yes please